src/content/docs/more-integrations/open-source-telemetry-integrations/airflow/monitoring-airflow-ot.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Monitor Airflow with OpenTelemetry and New Relic 
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Airflow
+  - Quickstart
+metaDescription: Here is a simple example to setup Airflow OpenTelemetry metrics to send data to New Relic.
+redirects:
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/airflow/monitoring-airflow-ot
+---
+
+import opentelemetryAirflow01 from 'images/opentelemetry_screenshot_airflow_01.webp'
+import opentelemetryAirflow02 from 'images/opentelemetry_screenshot_airflow_02.webp'
+
+Monitor Airflow with [OpenTelemetry Metrics](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#setup-opentelemetry) by visualizing tasks, operators, and DAG executions as metrics.
+
+<img
+  title="Screenshot showing sample Airflow DAG runs in New Relic"
+  alt="Screenshot showing sample Airflow DAG runs in New Relic"
+  src={opentelemetryAirflow01}
+/>
+
+## Prerequisites [#prerequisites]
+
+You need to first install the Airflow package with OpenTelemetry using the `otel` extra. This can be done in one of several ways, depending on how you deploy Airflow:
+
+### Installing from PyPi [#install-pypi]
+
+1. Follow the installation instructions from [Airflow's Documentation](https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html).
+2. When installing with pip, add the `otel` extra to the command.
+  a. eg. `pip install "apache-airflow[otel]"`
+
+### Installing from Docker [#install-docker]
+
+1. Setup the [Airflow Docker image](https://airflow.apache.org/docs/docker-stack/index.html) using instructions from [Airflow's Documentation](https://airflow.apache.org/docs/docker-stack/index.html).
+2. Extend the prebuilt image by using a Dockerfile to install the `otel` extra. You can replace the latest tag with your desired version of the Airflow image.
+
+```
+FROM apache/airflow:latest
+RUN pip install --no-cache-dir "apache-airflow[otel]==${AIRFLOW_VERSION}"
```

Comment:
@TimPansino Do they need to replace `AIRFLOW_VERSION`, or `{AIRFLOW_VERSION}`? Can it also be `latest`?

Reply:
It can't be latest, they've got really complex dependencies that initially get installed with a constraints file. Has to match the installed version.

Conveniently `AIRFLOW_VERSION` is baked into the image as an environment variable so there's no need to replace it.

Reply:
I made a change to this section, see if that's more clear.